nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3494,6 +3516,14 @@ Providing three total network interfaces, including  `nifi.web.http.network.inte
 |`nifi.web.https.host`|The HTTPS host. It is blank by default.
 |`nifi.web.https.port`|The HTTPS port. It is blank by default. When configuring NiFi to run securely, this port should be configured.
 |`nifi.web.https.port.forwarding`|Same as `nifi.web.http.port.forwarding`, but with HTTPS for secure communication. It is blank by default.
+|`nifi.web.https.includeciphersuites`|Cipher suites used to initialize the SSLContext of the Jetty https port.  If unspecified, the runtime SSLContext defaults are used.
```

Comment:
Looks like this property name needs to be updated, also recommend changing `https` to uppercase `HTTPS`:
```suggestion
|`nifi.web.https.ciphersuites.include`|Cipher suites used to initialize the SSLContext of the Jetty HTTPS port.  If unspecified, the runtime SSLContext defaults are used.
```